cmds/boot/boot/boot.go

```diff
@@ -469,8 +418,9 @@ func Localboot() error {
 		if *dryRun {
 			continue
 		}
-		if err := kexec.Reboot(); err != nil {
-			log.Printf("Kexec Reboot %v failed, %v. Sorry", u, err)
+
+		if err := boot.Execute(); err != nil {
+			log.Printf("kexec of %v failed: %v", u, err)
 		}
 		return nil
```

Comment:
I think there are lots of ways to return an error and we're just returning nil. We should probably return a real error or just drop the error return (or make a chan of errors?)

Reply:
in https://github.com/JulienVdG/tastevin I did handle similar stuff by joining all the error message before returning (having all errors and not only the first one helps especially when you wait 5 minutes to get the results)... or maybe we can do better using new go 1.13 error stuff

Reply:
I did just put a TODO for now so I won't forget during the rewrite